app/keepers/keepers.go

```diff
@@ -473,7 +473,8 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		AddRoute(txfeestypes.RouterKey, txfees.NewUpdateFeeTokenProposalHandler(*appKeepers.TxFeesKeeper)).
 		AddRoute(superfluidtypes.RouterKey, superfluid.NewSuperfluidProposalHandler(*appKeepers.SuperfluidKeeper, *appKeepers.EpochsKeeper, *appKeepers.GAMMKeeper)).
 		AddRoute(protorevtypes.RouterKey, protorev.NewProtoRevProposalHandler(*appKeepers.ProtoRevKeeper)).
-		AddRoute(gammtypes.RouterKey, gamm.NewMigrationRecordHandler(*appKeepers.GAMMKeeper))
+		AddRoute(gammtypes.RouterKey, gamm.NewMigrationRecordHandler(*appKeepers.GAMMKeeper)).
+		AddRoute(concentratedliquiditytypes.RouterKey, concentratedliquidity.NewConcentratedLiquidityProposalHandler(*appKeepers.ConcentratedLiquidityKeeper))
```

Comment:
Do we also not need to add the client handler as in https://github.com/osmosis-labs/osmosis/pull/4976/files#diff-95b3d541ce5c3238d75044a49770ab16c1bf1913ed0064a14897ed8474eba1b7R82?

Reply:
There are so many steps to adding a new message -__-, thanks for catching these.

Added codec, client, and cli here https://github.com/osmosis-labs/osmosis/pull/4982/commits/ae766e8ce75efba7721ffc658c3da9a4b484d2ae